backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/IInvoiceCandBL.java

```diff
@@ -407,4 +407,8 @@ void setInvoicingErrorAndSave(
 	InvoiceCandidatesAmtSelectionSummary calculateAmtSelectionSummary(@Nullable String extraWhereClause);
 
 	void setAsyncBatch(InvoiceCandidateId invoiceCandidateId, AsyncBatchId asyncBatchId);
+
+	Quantity getOrderedQtyStockUOM(I_C_Invoice_Candidate ic);
```

Comment:
```suggestion
	Quantity getQtyOrderedStockUOM(I_C_Invoice_Candidate ic);
```

..this is the more common naming convention..